static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-on-the-moon.js

```diff
@@ -0,0 +1,234 @@
+                } else {
+                    return false;
+                }
+            };
+            var userHasNeverContributed = !cookies.get('gu.contributions.contrib-timestamp');
+            var worksWellWithPageTemplate = (config.page.contentType === 'Article') && !config.page.isMinuteArticle; // may render badly on other types
+            return userHasNeverContributed && commercialFeatures.canReasonablyAskForMoney && worksWellWithPageTemplate && tagsMatch();
+        };
+
+        var makeEvent = (function(name) {
+            return this.id + ':' + name;
+        }).bind(this);
+
+        function makeUrl(urlPrefix, intcmp) {
+            return urlPrefix + 'INTCMP=' + intcmp;
+        }
+
+        var contributeUrlSuffix = 'co_au_epic_end_of_year_2016';
+        var membershipUrlSuffix = 'gdnwb_copts_mem_epic_end_of_year_2016';
+
+        var epicViewedEvent = makeEvent('view');
+        var epicInsertedEvent = makeEvent('insert');
+
+        var membershipUrl = 'https://membership.theguardian.com/supporter?';
+        var contributeUrl = 'https://contribute.theguardian.com/?';
+
+        var cta = {
+            cta1: 'Become a Supporter',
+            cta2: 'Make a contribution',
+            url1: makeUrl(membershipUrl, membershipUrlSuffix),
+            url2:  makeUrl(contributeUrl, contributeUrlSuffix),
+            hidden: ''
+        };
+
+        var componentWriter = function (component) {
+            ajax({
+                url: config.page.ajaxUrl + '/geolocation',
+                method: 'GET',
+                contentType: 'application/json',
+                crossOrigin: true
+            }).then(function (resp) {
+                if(resp.country === 'AU') {
+                    fastdom.write(function () {
+                        var submetaElement = $('.submeta');
+                        if (submetaElement.length > 0) {
+                            component.insertBefore(submetaElement);
+                            $('.contributions__epic').each(function (element) {
+                                // top offset of 18 ensures view only counts when half of element is on screen
+                                var elementInview = ElementInview(element, window, {top: 18});
+                                elementInview.on('firstview', function () {
+                                    mediator.emit(epicViewedEvent, component);
+                                });
+                            });
+                            mediator.emit(epicInsertedEvent, component);
+                        }
+                    });
+                }
+            });
+        };
+
+        function registerInsertionListener(track) {
+            mediator.on(epicInsertedEvent, track);
+        }
+
+        function registerViewListener(complete) {
+            mediator.on(epicViewedEvent, complete);
+        }
+
+        var canBeDisplayed = function() {
```

Comment:
This function is not necessary, all the logic around whether the test can run is in the canRun function on line 54.